docroot/modules/custom/sitenow_events/includes/sitenow_events.teaser.inc

```diff
@@ -81,8 +81,14 @@ function template_preprocess_sitenow_events_teaser(array &$variables) {
     'all_day' => $variables['event']['instance_all_day'],
   ]);
 
-  if (isset($variables['event']['description_text'])) {
-    $variables['event']['description_text'] = Xss::filter($variables['event']['description_text']);
-    $variables['event']['description_text'] = Unicode::truncate($variables['event']['description_text'], 500, TRUE, TRUE);
+  if (isset($variables['event']['description'])) {
+    // Limit what html is allowed.
+    $allowed_tags = ['a', 'strong', 'em'];
+    $variables['event']['description'] = Xss::filter($variables['event']['description'], $allowed_tags);
+
+    // Use smart_trim's helper to safely truncate without breaking HTML.
+    $truncate = new TruncateHTML();
+    $trim_suffix = '...';
+    $variables['event']['description'] = $truncate->truncateChars($variables['event']['description'], 400, $trim_suffix);
```

Comment:
Was the change from 500 characters to 400 intentional?

Reply:
Good point! I think I forgot to commit that change back.

Reply:
I changed it back to 500, are we good to merge this in?

Reply:
I think so!